homework/content.js

```diff
@@ -27,11 +30,12 @@
       p.textContent = passage;
       buff.appendChild(p);
     });
-    _wrapper.appendChild(h);
-    _wrapper.appendChild(buff);
+    this._wrapper.appendChild(h);
+    this._wrapper.appendChild(buff);
   }
 
   document.addEventListener('DOMContentLoaded', function(event) {
```

Comment:
Instead of keeping this, you should move all initialization lines into one start up method bind to this event, rather than two.